scikits/odes/sundials/cvode.pyx

```diff
@@ -678,7 +747,14 @@ cdef class CVODE:
             'max_nonlin_iters':
                 default = 0,
             'nonlin_conv_coef':
-                default = 0.
+                default = 0,
+            'old_api':
+                Values: True (default), False
```

Comment:
If we force new api for some options, I would make default old_api False, with here comment that for old code, old_api=True must be set in the code.